src/commands/identity/enableSystemIdentity.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizard, type ExecuteActivityContext, type IActionContext } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../localize";
+import { type SystemIdentityTreeItemBase } from "../../tree/remoteProject/SystemIdentityTreeItemBase";
+import { type SlotTreeItem } from "../../tree/SlotTreeItem";
+import { createActivityContext } from "../../utils/activityUtils";
+import { EnableSystemIdentityAssignStep } from "./EnableSystemIdentityStep";
+import { type ManagedIdentityAssignContext } from "./ManagedIdentityAssignContext";
+
+export async function enableSystemIdentity(context: IActionContext, node: SystemIdentityTreeItemBase): Promise<undefined> {
+    const grandparentNode = node.parent.parent as SlotTreeItem;
```

Comment:
With my other suggestion, I think you might be able to avoid having to typecast this value